src/System.Management.Automation/resources/CommandBaseStrings.resx

```diff
@@ -156,6 +156,15 @@
   <data name="PauseHelpMessage" xml:space="preserve">
     <value>Pause the current pipeline and return to the command prompt. Type "{0}" to resume the pipeline.</value>
   </data>
+  <data name="ProgramFailedToComplete" xml:space="preserve">
```

Comment:
How about renamed this string to `ProgramEndsWithNonZeroCode`? It seems clearer to me, and we can also remove the comments then.
Also, it's not guaranteed the program failed just because it emits non-zero exit code.